src/VisualStudio.Roslyn.SDK/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Control/SyntaxKindHelper.cs

```diff
@@ -15,9 +15,9 @@ public static string GetKind(this SyntaxNodeOrToken nodeOrToken)
         {
             var kind = string.Empty;
 
-            if (nodeOrToken.IsNode)
+            if (nodeOrToken.IsNode && nodeOrToken.AsNode() is SyntaxNode node)
```

Comment:
Two changes suggested here:

1. The `IsNode` check is not necessary since `AsNode()` is already checking the same thing
2. The `var` syntax here is `{ }`

```suggestion
            if (nodeOrToken.AsNode() is { } node)
```